warehouse/schema/schema.go

```diff
@@ -375,9 +339,9 @@ func (sh *Schema) removeDeprecatedColumns(schema model.Schema) {
 	}
 }
 
-// hasSchemaChanged compares the localSchema with the schemaInWarehouse
-func (sh *Schema) hasSchemaChanged(localSchema model.Schema) bool {
-	return !reflect.DeepEqual(localSchema, sh.schemaInWarehouse)
+// HasSchemaChanged compares the localSchema with the schemaInWarehouse
+func (sh *Schema) HasSchemaChanged(schema model.Schema) bool {
+	return !reflect.DeepEqual(schema, sh.localSchema)
```

Comment:
@cisse21 Is the comment correct now? It will compare the schema with localSchema now?